plugins/connectrpc/query-es/v1.4.1/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "plugins-connectrpc-query-es",
+  "version": "1.4.1",
+  "dependencies": {
+    "@connectrpc/protoc-gen-connect-query": "1.4.1"
+  },
+  "devDependencies": {
+    "esbuild": "^0.21.3"
```

Comment:
For node plugins using esbuild, I'd like to figure out how to keep these dependencies upgraded.